stacks/core/addons/tls.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.7'
+
+# This files contains the necessary traefik configuration to enable the HTTP services to communicate securely.
+# Details: https://docs.traefik.io/https/tls/
+
+services:
+  babeltower:
+    deploy:
+      labels:
+        - traefik.http.routers.babeltower.entrypoints=websecure
+        - traefik.http.routers.babeltower.tls=true
+        - traefik.http.routers.babeltower.tls.certresolver=https
+
+  storage:
+    deploy:
+      labels:
+        - traefik.http.routers.storage.entrypoints=websecure
+        - traefik.http.routers.storage.tls=true
+        - traefik.http.routers.storage.tls.certresolver=https
+
+  traefik:
```

Comment:
TODO: add http > https redirecting.
It's already done on production, only copy/paste here.

Reply:
Added a 'TODO' in order to investigate better how to integrate a global redirect mechanism and avoid polluting the stack with repeated configuration.